directives/questionnaire.py

```diff
@@ -433,6 +438,9 @@ def input_type(self):
 
 class MultipleChoice(Choice):
     ''' Lists options for picking all the correct ones. '''
+    # inherit QuestionMixin options and add a key
```

Comment:
Add an empty line after the docstring `''' Lists options .... '''` and start your new comment with the capital letter `I` and end the sentence with the period `.`.